blocks/identity-block/components/FormInputField/index.jsx

```diff
@@ -74,6 +75,7 @@ const FormInputField = ({
           ...(!valid ? ['xpmedia-form-field-input--error'] : []),
         ].join(' ')}
         id={inputId}
+        name={name}
```

Comment:
was this change deliberate? with field parameters

Reply:
Yeah.  Name is required since it's used in the id for the field group.
```
  const infoId = `id_${name}_error`;
  const inputId = `id_${name}_input`;
```